packages/docs/site/docs/01-start-here/01-index.md

```diff
@@ -9,8 +9,9 @@ slug: /start-here
 
 The documentation consists of two major sections:
 
--   [Documentation](./01-index.md) – Introduction, concepts, and guides
--   [API reference](/api) – All the APIs exposed by WordPress Playground
+-   [Overview] (./02-overview.md) - What is it and how it works?
```

Comment:
Notice how this now says "The documentation consists of two major sections: Overview, Documentation, API Reference." Maybe this would work better?

```
👋 Hi! Welcome to WordPress Playground documentation. Playground is an online tool to experiment and learn about WordPress – learn more in the [overview section](./02-overview.md).

The documentation consists of two major sections:
```

Reply:
Ah, sorry I missed it my bad. 😃

Reply:
All good!